CONTRIBUTING.md

```diff
@@ -23,14 +24,22 @@ If you want to fix a typo or otherwise improve on existing content, follow a sim
 
 Use language consistent with [TERMINOLOGY](https://github.com/solana-foundation/developer-content/blob/main/docs/terminology.md). In particular:
 
-- Use 'secret key' rather than 'private key'. 
+- Use 'secret key' rather than 'private key'.
 - Use 'wallet app' for software. 'wallet' for the address that holds value.
-- PDAs are not public keys. It is not possible to have a public key without a private key. A public key is derived from a private key, and it is not possible to generate a public key without first generating a private key.
-- Do not use the term 'token account'. The term 'token account' is vague, and could be interpreted as either 'associated token account' or 'token mint account', or some new thing that doesn't exist. Instead use 'associated token account' or 'token mint account' depending on what is being referred to.
+- Use 'SOL' rather than 'Sol' to refer to Solana's native token
+- PDAs are not public keys. It is not possible to have a public key without a secret key. A public key is derived from a secret key, and it is not possible to generate a public key without first generating a secret key.
+- Avoid using the term 'token account' if something more specific will suffice. [The term 'token account' can refer to many types of Solana account that have the bytecode format of token accounts](https://solana.stackexchange.com/questions/7507/what-is-the-difference-between-a-token-account-and-an-associated-token-account/7521#7521). Instead use a more specific term like 'associated token account' or 'token mint account' depending on what is being referred to if possible.
```

Comment:
I softened this language after talking to Jordan Sexton about these definitions - see the Solana Stack Overflow link.

Reply:
Ah thank you. This was actually something I was going to talk to you about. We're aligned on the most recent changes you've added here though